src/test/java/StylesCheck.java

```diff
@@ -0,0 +1,128 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.Color;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.nio.channels.WritableByteChannel;
+import java.time.Duration;
+
+public class StylesCheck {
+
+    private WebDriver driver;
+
+    @BeforeSuite
+    public void setupDriverManager() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeTest
```

Comment:
Use `@BeforeMethod` and `@AfterMethod` annotations here also. Even there is no login logic here, it is good to have a clean setup in each test.